### PR TITLE
Add naive implementations of 2 resources-related methods.

### PR DESCRIPTION
**Pull request by ericsnowcurrently**
These are the 2 Client methods that Juju needs.

This patch also adds a NewCharmStoreFromCilent() function that Juju will use.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmrepo/153/
Test FAILed.

**Comment by ericsnowcurrently** [comment]
Not sure who runs the CI server, but it looks like the problem is with downloading godeps...

File: csclient/csclient.go [hunk 1]
```diff
@@ -100,6 +101,10 @@ func (c *Client) SetHTTPHeader(header http.Header) {
 	c.header = header
 }
 
+// TODO(ericsnow) Is the returned hash the base64-encoded
+// representation? If so then the doc comment should be clear
+// about that.
```
Comment (frankban) [inline csclient/csclient.go]: The returned hash is fmt.Sprintf("%x", hash.Sum(nil))

**Comment by frankban** [comment]
We are working on fixing the CI issue.
The branch looks good :+1:

**Comment by urosj** [comment]
We have a client already and it has GetArchive 
Not too keen on this PR, it acts as it works, but it's just a stub.

**Comment by ericsnowcurrently** [comment]
I'll get in touch so we can get on the same page.

**Comment by bac** [comment]
CI should work now if you commit a new version.  Here is one I ran by hand:
http://ci.jujugui.org:8080/job/charmrepo/156/

**Comment by ericsnowcurrently** [comment]
thanks

**Comment by ericsnowcurrently** [comment]
I'm going to separate the clean-up bits from the resources bits into a new PR and wait for more discussion before tackling the resuorces part again. :)

**Pull request closed without merging**